cost/scheduled_reports/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.0
```

Comment:
version should be 1.20.

Reply:
I have addressed the comment please review the changes